client/components/select-dropdown/index.jsx

```diff
@@ -207,12 +207,20 @@ class SelectDropdown extends Component {
 			} );
 		}
 
-		let dropdownClassName = classNames( dropdownClasses );
-		let selectedText = this.props.selectedText
-			? this.props.selectedText
-			: result( find(
-				this.props.options, { value: this.state.selected }
+		const dropdownClassName = classNames( dropdownClasses );
+
+		let selectedText = this.props.selectedText;
```

Comment:
> Is it ok if I simply push a new commit on this branch?

Yes please do. 😄  We also prefer to rebase branches with master (rather than merge) to keep history clean. We'll of course make sure you'll get credit for the fix, so let's try to squash the changes on this branch to a single commit, so @alisterscott can do a merge commit into master.

See also our [contributing doc](https://github.com/Automattic/wp-calypso/blob/master/.github/CONTRIBUTING.md#lifecycle-of-a-pull-request)